sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -183,21 +184,21 @@ AZ_NODISCARD az_result az_iot_hub_client_sas_get_signature(
  * @param[in] client The #az_iot_hub_client to use for this call.
  * @param[in] base64_hmac_sha256_signature The Base64 encoded value of the HMAC-SHA256(signature,
  *                                         SharedAccessKey). The signature is obtained by using
- *                                         #az_iot_hub_client_sas_get_signature.
+ *                                         az_iot_hub_client_sas_get_signature().

```

Comment:
```suggestion
 *                                         #az_iot_hub_client_sas_get_signature().

```

Reply:
The `#` isn't needed if it is a function. Doxygen looks for words followed by `()` in it's documented function "list". So this will render a link to the proper docs.